### PR TITLE
Refactor hard-coded configuration strings into const variables to improve code readability and extensibility

### PR DESCRIPTION
**Pull request by panyuenlau**
| Q               | A      |
| --------------- | ------ |
| Bug fix?        | no|
| New feature?    | no |
| API breaks?     | no |
| Deprecations?   | no |
| License         | Apache 2.0 |

### What's in this PR?

Noticed that the current implementation sets a bunch of hard-coded strings for Kafka and Cruise Control configurations, and I think it's more readable and bug-prune for all the Kafka and Cruise Control configurations to be grouped together in a file so we can use them in the source code

> Note: didn't really update the test because I think the tests are fine to use hard-coded strings as they are now

### Why?

Concepts that are closely related should be kept vertically close to each other

### Test

Test with both internal and external listeners with plaintext, didn't get to test all the security protocols - but triple compared the hard-coded strings with the newly created consts

### CODE REVIEW

**Comment by pregnor** [comment]
Friendly ping, when to expect this to be finished. (No pressure just informational.)

**Comment by panyuenlau** [comment]
> Friendly ping, when to expect this to be finished. (No pressure just informational.)

The refactoring itself is pretty much done, the main purpose for the refactoring is to prepare for the KRaft support so things won't look even messier when we add the KRaft. 

I will just need to do some manual testings to ensure I'm not breaking anythibg here

**Review by pregnor — APPROVED** [review]
LGTM, thanks for this.

File: pkg/util/kafka/const.go [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+package kafka
+
+const (
+	// AllBrokerServiceTemplate template for Kafka all broker service
+	AllBrokerServiceTemplate = "%s-all-broker"
+	// HeadlessServiceTemplate template for Kafka headless service
+	HeadlessServiceTemplate = "%s-headless"
+	// NodePortServiceTemplate template for Kafka nodeport service
+	NodePortServiceTemplate = "%s-%d-%s"
```
[the remark this answers is not in the document]
Comment (panyuenlau) [reply pkg/util/kafka/const.go]: Make sense, these template consts have been moved to `template.go` now, thx for the suggestion!

**Review by Kuvesz — APPROVED** [review]
Looks good, thanks a lot! :)

**Review by bartam1 — APPROVED** [review]

**Pull request merged**